bc_obps/reporting/models/emission_category.py

```diff
@@ -25,3 +27,24 @@ class Meta:
             "This table contains the set of emission categories that greenhouse gas emissions can be counted under."
         )
         db_table = 'erc"."emission_category'
+
+    class Rls:
```

Comment:
https://docs.djangoproject.com/en/5.1/topics/db/models/#meta-inheritance
We could make this an `rls = ...` property on the existing Meta class by extending it